python/paddle/fluid/layers/nn.py

```diff
@@ -352,6 +353,38 @@ def fc(input,
     return helper.append_activation(pre_activation)
 
 
+def center_loss(input,
+                labels,
+                num_classes,
+                alpha,
+                param_attr,
```

Comment:
what is this param_attr used for ?

Reply:
param_attr is used for initialize the centers use xavier
center_loss=fluid.layers.center_loss(input=pool_out,
                labels=label,
                num_classes=1000,
                alpha=lr_center,
                param_attr=fluid.initializer.Xavier(uniform=False),
                update_center=True)